modules/unsupported/arcgis-rest/pom.xml

```diff
@@ -121,6 +113,11 @@
             <version>23-SNAPSHOT</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
Better to create a profile for this that is activated by the JDK in use, that way you won't run into conflicts when running on JDK8.

see for example https://github.com/geoserver/geofence/pull/154 and https://github.com/geoserver/geofence/commit/67c982239c59f080444830b6000dd0421d2